client/docs/py-primer.html

```diff
@@ -0,0 +1,422 @@
+<html>
```

Comment:
Damn, you outmaneuvered me on this -- I didn't notice you added it. Nice work catching this! Make sure to delete *-light and *-dark from the primers if you haven't already.

Reply:
Yup, I've deleted them! Eventually, will we still need the primer (don't see the link on the design doc anymore)?